src/environments/environment.development.ts

```diff
@@ -0,0 +1,5 @@
+export const environment = {
+  production: false,
+  abuseIpUrl: 'https://api.abuseipdb.com/api/v2',
+  abuseIpApiKey: '',
```

Comment:
All envs are declared in a typescript file? Do you know any way to declare the key privately without commiting it?

Also, this is a frontend application. Does Angular have any way to not expose the key? Just asking out of curiosity. This project will be just an example.